services/worker/src/worker/job_runners/split/opt_in_out_urls_scan_from_streaming.py

```diff
@@ -157,8 +157,7 @@ def compute_opt_in_out_urls_scan_response(
         urls_count = sum(
             1
             for row in first_rows
-            if isinstance(row["row"].get(string_column), str)
-            and (row["row"][string_column].startswith("https://") or row["row"][string_column].startswith("http://"))
+            if isinstance(row["row"].get(string_column), str) and is_image_url(text=row["row"][string_column])
         )
         if urls_count and urls_count / len(first_rows) > 0.5:
```

Comment:
I think you can reduce the ratio, in case datasets have some URLs without image extension
```suggestion
        if urls_count and urls_count / len(first_rows) > 0.3:
```